lib/msf/core/exploit/remote/kerberos/ticket_converter.rb

```diff
@@ -0,0 +1,78 @@
+module Msf::Exploit::Remote::Kerberos
+  module TicketConverter
+    def self.kirbi_to_ccache(krb_cred)
```

Comment:
Looks like a lot of these methods are missing the missing type information 👀

Reply:
Think I got them all now cheers